meta-mender-core/recipes-bsp/grub/grub-efi_%.bbappend

```diff
@@ -16,14 +35,6 @@ CONFFILES_${PN}_remove_mender-grub = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'efi-secure-boot', '${EFI_BOOT_PATH}/efi-secure-boot.inc', '', d)} \
 "
 
-do_install_append_class-target() {
-    if "${@bb.utils.contains('DISTRO_FEATURES', 'mender-install', 'true', 'false', d)}"; then
-        install -d ${D}/${MENDER_BOOT_PART_MOUNT_LOCATION}
-        mv ${D}/boot/EFI ${D}/${MENDER_BOOT_PART_MOUNT_LOCATION}
-        rmdir ${D}/boot || true
-    fi
-}
```

Comment:
@drewmoseley: Note in particular that I have removed both of these two very similar fixes that you made. The key is the assignment of `EFI_PREFIX`, which you can find elsewhere in this PR. This means that the original recipe actually puts the file in the right place, and this part is not needed, nor is it valid, anymore. But if you have things depending on this it probably deserves a test round.